### PR TITLE
fix(parser): Fixed "Tradable" property conditions

### LINKED ISSUE #5: Clean up tradable markers
There's a very high chance that not all tradable/untradable items are identified correctly. *Most* should be, but still requires some tweaking. The relevant code snippet can be found at `scraper.addTradable`.

Just a bunch of things I noticed so far:
- [ ] Arcanes
- [x] Excal/Lato/Skana Prime shouldn't be tradable
- [ ] Non-refined ore and other resources shouldn't be tradable
- [ ] Planet nodes shouldn't be considered at all

Please add your own below if you find any other issues.

### PR DESCRIPTION
**Pull request by FabienPaugois**
### What did you fix? (provide a description or issue closes statement)
fix majority of Issue \#5 problems

closes #5 

---

### Reproduction steps
1. Fetched all items from API
2. Filter to get tradable items
3. Noticed that some were untradable but shouldn't be untradable, and same for tradable

---

### Evidence/screenshot/link to line

You can see my fix [on this line](#line-number-467) for where the new data exists

### Considerations
- Does this contain a new dependency? **[No]**
- Does this introduce opinionated data formatting or manual data entry? **[Yes]**
- Does this pr include the updated data files in a separate commit that can be reverted for a clean code-only pr? **[No]**
- Have I run the linter through `npm run lint`? **[Yes]**
- Is is a bug fix, feature request, or enhancement? **[Bug Fix + Enhancement]**

### CODE REVIEW

**Comment by TobiTenno** [comment]
Still seeing failures

File: build/parser.js [hunk 1]
```diff
@@ -465,13 +465,14 @@ class Parser {
    * Limit items to tradable/untradable if specified.
    */
   addTradable (item) {
-    const tradableTypes = ['Upgrades', 'Fish', 'Key', 'Focus Lens', 'Relic', 'Rifle Mod',
+    const tradableTypes = ['Arcane', 'Fish', 'Focus Lens', 'Relic', 'Rifle Mod',
```
Comment (TobiTenno) [inline build/parser.js:468]: why'd you remove Upgrades?

File: build/parser.js [hunk 2]
```diff
@@ -465,13 +465,14 @@ class Parser {
...
       'Melee Mod']
-    const untradableTypes = ['Skin', 'Medallion', 'Extractor', 'Pets', 'Ship Decoration']
-    const tradableRegex = /(Prime|Vandal|Wraith|Rakta|Synoid|Sancti|Vaykor|Telos|Secura)/i
-    const untradableRegex = /(Glyph|Mandachord|Greater.*Lens|Sugatra)/i
-    const notFiltered = !untradableTypes.includes(item.type) && !item.name.match(untradableRegex)
+    const untradableTypes = ['Skin', 'Medallion', 'Key', 'Extractor', 'Pets', 'Ship Decoration',
+      'Glyph', 'Sigil', 'Fur Color', 'Syandana', 'Fur Pattern', 'Color Palette', 'Node', 'Exalted Weapon']
+    const tradableRegex = /(Prime|Vandal|Wraith|Rakta|Synoid|Sancti|Vaykor|Telos|Secura|Ayatan|Prisma)/i
+    const untradableRegex = /(Glyph|Mandachord|Greater.*Lens|Sugatra|\[|SentinelWeapons|Toroid|Bait|([A-Za-z]+ (Relic)))/i
+    const notFiltered = !untradableTypes.includes(item.type) && !item.name.match(untradableRegex) && !item.uniqueName.match(untradableRegex) && (item.hasOwnProperty('productCategory') ? !item.productCategory.match(/(SpecialItems)/) : true)
```
Comment (TobiTenno) [inline build/parser.js:475]: could you wrap some of the lines down, like pull the options in the ternary so it's readable? i probably need to add an editorconfig with line length so this is enforced if it's not covered in the existing linting

File: build/parser.js (hunk 1 above)
Comment (FabienPaugois) [reply build/parser.js:468]: I noticed that it wasn't doing anthing, but I can have it back if you want.
Removing it didn't change the JSON Data when I ran the `npm run build`

Comment (FabienPaugois) [reply build/parser.js:468]: Can confirm after further testing that I got the exact same results with and without 'upgrades'

Comment (TobiTenno) [reply build/parser.js]: okiedoke. feel free to leave it out.

**Review by TobiTenno — APPROVED** [review]
seems good to me

**Pull request merged**